rfd/0120/README.md

```diff
@@ -27,13 +27,16 @@ network.
 
 ## Proposed Solution
 
+XXX KEBE SAYS introductory text goes here regarding which *APIs change,
+etc. etc.... XXX
+
 ### NAPI changes
 
 #### CreateRouterObject (POST /routers)
 
 Creating a Router Object simply involves listing a number of NAPI network
 objects one wishes to join together.  For now, these network objects MUST be
-fabrics.
+fabrics, and MUST be owned by the same owner_uuid that submits the request.
 
 ##### input
```

Comment:
Probably need the owner_uuid in the payload as well, since NAPI will not have the context of the caller and rely on CloudAPI (or the operator who makes the direct NAPI request) to supply the owner_uuid.

Reply:
Will fix by adding owner_uuid into the required inputs.